lib/miq_automation_engine/service_models/miq_ae_service_network_router.rb

```diff
@@ -8,8 +8,5 @@ class MiqAeServiceNetworkRouter < MiqAeServiceModelBase
     expose :network_ports,         :association => true
     expose :vms,                   :association => true
     expose :private_networks,      :association => true
-
-    expose :raw_update_network_router
-    expose :raw_delete_network_router
```

Comment:
@blomquisg In this case I removed them because the underlying methods are no longer there. When I was adding the new methods for this PR I realized that a prior change had removed these two without removing the expose calls.